RZBluetooth/RZBPeripheral.m

```diff
@@ -138,9 +144,10 @@ - (void)enableNotifyForCharacteristicUUID:(CBUUID *)characteristicUUID
     RZBNotifyCharacteristicCommand *cmd = [[RZBNotifyCharacteristicCommand alloc] initWithUUIDPath:path];
     cmd.notify = YES;
     [cmd addCallbackBlock:^(CBCharacteristic *characteristic, NSError *error) {
-        if (characteristic != nil) {
```

Comment:
I'd prefer to keep this conditional the way it was to only register the notify block if the characteristic is found.